src/config.cc

```diff
@@ -7,160 +7,161 @@ using helpers::debugf;
-  }
 
-  if (!gotPort) {
-    debugf("Config::Validate", "No port specified in configuration file.\n");
-    return false;
-  }
-
-  return true;
+    return true;
 }
 
 unsigned short Config::GetPort(const NginxConfig *conf)
 {
-  for (auto const& stmt : conf->statements_) {
-    if (stmt->tokens_.size() == 2 && stmt->tokens_[0] == "port") {
-      // guaranteed to be within [1,65535] if Valid passed for NginxConfig
-      return (unsigned short) strtol(stmt->tokens_[1].c_str(), nullptr, 10);
+    for (auto const& stmt : conf->statements_) {
+        if (stmt->tokens_.size() == 2 && stmt->tokens_[0] == "port") {
+            // guaranteed to be within [1,65535] if Valid passed for NginxConfig
+            return (unsigned short) strtol(stmt->tokens_[1].c_str(), nullptr, 10);
+        }
     }
-  }
-  debugf("Config::GetPort", "Failed to get port from config. This should "
-      "never happen.\n");
-  return 0;
+    debugf("Config::GetPort", "Failed to get port from config. This should "
+            "never happen.\n");
+    return 0;
 }
 
 
 bool Config::GetHandlerInfo(const NginxConfig *conf,
-    const Request *req, NginxConfigStatement *&info)
+        const Request *req, NginxConfigStatement *&info)
 {
-  /*
-   * Look over req and conf to determine handler with longest matching prefix.
-   */
-  std::string longest_prefix;
-  std::string uri = req->uri();
-  for (auto const& stmt : conf->statements_) {
-    if (stmt->tokens_[0] == "path") {
-      if (uri.find(stmt->tokens_[1]) == 0 &&
-          longest_prefix.size() < stmt->tokens_[1].size()) {
-        longest_prefix = stmt->tokens_[1];
-        info = stmt.get();
-      }
+    /*
+     * Look over req and conf to determine handler with longest matching prefix.
+     */
+    std::string longest_prefix;
+    std::string uri = req->uri();
+    for (auto const& stmt : conf->statements_) {
+        if (stmt->tokens_[0] == "path") {
+            if (uri.find(stmt->tokens_[1]) == 0 &&
+                    longest_prefix.size() < stmt->tokens_[1].size()) {
+                longest_prefix = stmt->tokens_[1];
+                info = stmt.get();
+            }
+        }
     }
-  }
 
-  /*
-   * Nothing matches the given URI, so return false.
-   */
-  if (longest_prefix.size() == 0) {
-    return false;
-  }
+    /*
+     * Nothing matches the given URI, so return false.
+     */
+    if (longest_prefix.size() == 0) {
+        return false;
+    }
 
-  return true;
+    return true;
 }
 
 void Config::GetAllHandlerInfo(const NginxConfig *conf,
-    std::list<NginxConfigStatement const *> &info)
+        std::list<NginxConfigStatement const *> &info)
 {
-  for (auto const& stmt : conf->statements_) {
-    if (stmt->tokens_[0] == "path" || stmt->tokens_[0] == "default") {
-      info.push_back(stmt.get());
+    for (auto const& stmt : conf->statements_) {
+        if (stmt->tokens_[0] == "path" || stmt->tokens_[0] == "default") {
+            info.push_back(stmt.get());
+        }
     }
-  }
 }
```

Comment:
Not to fault you, of course. We may want to consider refactoring some of this in the future